### DISCUSSION TITLE
SDcard connector on backpanel

### DISCUSSION THREAD

**Discussion by atomicdog** [TheGuyDanish/CM4_MATX#70] [General]
I have a question about the SDIO wiring. Was it meant to be only for CM4Lite? 
The schematic has it wired to the pins for the CM4lite's access to external SDIO/eMMC. The manual states those pins are only for the CM4Lite.

**Comment by TheGuyDanish** [comment]
Like the IO Board, the SDIO wiring to expose a microSD card slot is intended for people using it with a CM4Lite. The pins that make up the SDIO, to my knowledge, are simply not driven on the non-lite SKUs. Thus having the slot there, even if you're using the board with a non-lite CM4, has no real downside. It's just a precaution for anyone who _is_ using it with a CM4Lite.

**Reply by atomicdog** [reply]
I think having it at the I/O shield location creates some confusion. People will assume it's meant for general purpose use.  
  Since the SD card for the CM4Lite probably will be used for the OS I don't think people will need or even want quick external access to it (like accidentally ejecting the card when running). Also, when developing software and you do need to remove the SD card I think you would already want to have the PC case open or the motherboard out of the case so you can troubleshoot/inspect hardware.   
 In other words I think an internal microSD card slot near the CM4 would be better.  It would also make easier signal routing and also simplify making future revision with an eMMC.
